hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/timeline/versioning/v2/LSMTimelineWriter.java

```diff
@@ -116,10 +117,27 @@ public static LSMTimelineWriter getInstance(HoodieWriteConfig config, TaskContex
   public void write(
       List<ActiveAction> activeActions,
       Option<Consumer<ActiveAction>> preWriteCallback,
-      Option<Consumer<Exception>> exceptionHandler) throws HoodieCommitException {
+      Option<Consumer<Exception>> exceptionHandler) {
     ValidationUtils.checkArgument(!activeActions.isEmpty(), "The instant actions to write should not be empty");
     StoragePath filePath = new StoragePath(this.archivePath,
         newFileName(activeActions.get(0).getInstantTime(), activeActions.get(activeActions.size() - 1).getInstantTime(), FILE_LAYER_ZERO));
+    try {
+      if (this.metaClient.getStorage().exists(filePath)) {
```

Comment:
How frequently the lsm writer is called? Little bit concerned about `exists` check on every call.